CrazyCanvas/Include/EventHandlers/AudioEffectHandler.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include "Physics/PhysicsEvents.h"
+
+struct ProjectileHitEvent;
+
+namespace LambdaEngine
+{
+	class ISoundEffect3D;
+}
+
+// AudioEffectHandler handles events that should cause audio effects to be played
```

Comment:
How should we think about SoundEffectHandler and AudioSystem?
One way is to say soundeffects that only requires PlayOnceAt. Which at the moment only is projectile hit.

Reply:
`AudioSystem` is part of the engine's API for playing sounds. `SoundEffectHandler` is a class specific to the game, that specializes in knowing what sounds to play during certain events. When it plays a sound, it uses the engine's sound API (`AudioSystem` or otherwise) to do so.